### PR TITLE
Make compatible with new NL permission system

### PR DESCRIPTION
**Pull request by Maxopoly**
Let's Bastion use the new functionality added with https://github.com/Civcraft/NameLayer/pull/191

Should be merged after the NameLayer PR is merged.

### CODE REVIEW

File: Bastion/pom.xml [hunk 1]
```diff
@@ -39,11 +39,12 @@
...
-      <groupId>vg.civcraft.mc.namelayer</groupId>
-      <artifactId>NameLayer</artifactId>
-      <version>2.5.42</version>
-      <scope>provided</scope>
-    </dependency>
+            <groupId>vg.civcraft.mc.namelayer</groupId>
+            <artifactId>NameLayer</artifactId>
+            <version>2.6</version>
+            <scope>system</scope>
+            <systemPath>C:\Users\Max\Desktop\civ plugins\Jars for dependencies\NameLayer-2.6.jar</systemPath>
```
Comment (ProgrammerDan) [inline Bastion/pom.xml]: Probably don't want this line here.

File: Bastion/src/isaac/bastion/Bastion.java [hunk 2]
```diff
@@ -88,5 +93,15 @@ public void removeGhostBlocks(){
...
 	}
+	
+	public void registerNameLayerPermissions() {
+		LinkedList <PlayerType> memberAndAbove = new LinkedList<PlayerType>();
+		memberAndAbove.add(PlayerType.MEMBERS);
+		memberAndAbove.add(PlayerType.MODS);
+		memberAndAbove.add(PlayerType.ADMINS);
+		memberAndAbove.add(PlayerType.OWNER);
+		PermissionType.registerPermission("BASTION_PEARL", memberAndAbove);
+		PermissionType.registerPermission("BASTION_PLACE", memberAndAbove);
```
Comment (ProgrammerDan) [inline Bastion/src/isaac/bastion/Bastion.java]: I thought previously block placement was MOD and up?

File: Bastion/pom.xml (hunk 1 above)
Comment (Maxopoly) [inline Bastion/pom.xml]: Yep, the build server definitely wouldn't like that

File: Bastion/src/isaac/bastion/Bastion.java (hunk 2 above)
Comment (Maxopoly) [inline Bastion/src/isaac/bastion/Bastion.java]: Was it? I have no idea, but I'm just gonna believe you that it was.

Comment (ProgrammerDan) [inline Bastion/src/isaac/bastion/Bastion.java]: Yeah, the isAccessible() call required BLOCKS permission which by default starts at MOD

**Comment by ProgrammerDan** [comment]
Other then noted looks good

**Pull request merged**